test/integration/parachain/staging/sequential/replace.test.ts

```diff
@@ -80,70 +81,91 @@ describe("replace", () => {
-            assert.equal(requestsList[0].btcHeight.toString(), firstMapEntry.value.btcHeight.toString());
-        }).timeout(1000000);
+            const membersFromListToExpect = requestsList.map(req => ({
+                btcAddress: req.btcAddress,
+                btcHeight: req.btcHeight,
+                amount: req.amount.toString()
+            }));
 
-        it("should fail vault replace request if not having enough tokens", async () => {
-            interBtcAPI.setAccount(vault_2);
+            const membersFromMapToCheck = Array.from(requestsMap.values()).map(req => ({
+                btcAddress: req.btcAddress,
+                btcHeight: req.btcHeight,
+                amount: req.amount.toString()
+            }));
 
-            // fetch tokens held by vault
-            const tokensInVault = await interBtcAPI.vaults.getIssuedAmount(
-                newAccountId(api, vault_2.address),
-                currencyIdToLiteral(vault_2_id.currencies.collateral)
-            );
+            expect(membersFromMapToCheck).to.deep.include.members(membersFromListToExpect);
+        }).timeout(2000000);
 
-            // make sure vault does not hold enough issued tokens to request a replace
-            const replaceAmount = dustValue.add(tokensInVault);
-            
-            try {
-                await interBtcAPI.replace.request(
-                    replaceAmount, 
-                    currencyIdToMonetaryCurrency(vault_3_id.currencies.collateral) as CollateralCurrency
+        it("should fail vault replace request if not having enough tokens", async () => {
+            for (const vault_2_id of vault_2_ids) {
+                const currencyTicker = currencyIdToMonetaryCurrency(vault_2_id.currencies.collateral).ticker;
+                interBtcAPI.setAccount(vault_2);
+    
+                // fetch tokens held by vault
+                const tokensInVault = await interBtcAPI.vaults.getIssuedAmount(
+                    newAccountId(api, vault_2.address),
+                    currencyIdToLiteral(vault_2_id.currencies.collateral)
                 );
-                assert.fail("Expected error to be thrown due to lack of issued tokens for vault, but call completed.");
-            } catch (e) {
-                assert.isTrue(e instanceof Error, "Expected replace request to fail with Error");
+    
+                // make sure vault does not hold enough issued tokens to request a replace
+                const replaceAmount = dustValue.add(tokensInVault);
+                
+                try {
+                    await interBtcAPI.replace.request(
+                        replaceAmount, 
+                        currencyIdToMonetaryCurrency(vault_2_id.currencies.collateral) as CollateralCurrency
+                    );
+                    assert.fail(`Expected error to be thrown due to lack of issued tokens
+                        for vault (collateral: ${currencyTicker}), but call completed.`);
```

Comment:
Would be cleaner to check that the Promise is rejected, also not blocking

Reply:
Also added to the follow-up task.
Thank you for the suggestion.